### DISCUSSION TITLE
Automate change log

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#393] [Ideas]
Issue titles from issues with "milestone highlight" could be automatically found and added to the change log.

**Comment by doubledh** [comment]
I can imagine implementing that as a got action, by using the github API to get current milestone, get issues from current milestone, and filter for milestone-highlight label

**Comment by doubledh** [comment]
I’m not 100% sure if the issue names will look good on the changelog, though. They might be too long, but they might also not be because titles aren’t usually light-novel-long

**Reply by ntseng** [reply]
I definitely haven't been writing issue titles to look like change log notes. Knowing to try would probably help a lot.

**Comment by doubledh** [comment]
so… what exactly constitutes “milestone highlight”, again? I was thinking about it again, and I would think it’s any changes that can:
- change what players can work with
- change UI (how to view/interact)
- rebalances that significantly (?) affect strategy… we wouldn’t rebalance if it wasn’t significant to en/discourage a strategy though, right?

I think that’s basically:
- content/bugs
- mechanics/bugs
- most reimplementations / rebalances (?)
- external docs/UI

and not:
- refactors
- dev tool
- internal docs
- library update

**Reply by ntseng** [reply]
I was defining it as "worthy of appearing in the patch notes", so given that the patch notes reads from the change log felt there was a good amount of overlap.
